source/io/PropertyNode.cpp

```diff
@@ -94,12 +111,25 @@ VIREO_FUNCTION_SIGNATUREV(PropertyNodeRead, PropertyNodeReadParamBlock)
     StringRef propertyName = _Param(propertyName);
     StaticTypeAndData *value = _ParamImmediate(value);
 
+    ExecutionContextRef exec = THREAD_EXEC();
+    VIClump* runningQueueElt = exec->_runningQueueElt;
+    VirtualInstrument* vi = runningQueueElt->OwningVI();
+    TypeManagerRef typeManager = vi->TheTypeManager();
```

Comment:
Instead of all this, you can get the TypeManager out of the StaticTypeAndData parm you already have.
value->_paramType->TheTypeManager()

Reply:
I wasn't sure if that was a recommended way to get. Great to know it is. I'll change it.